.buildkite/build_pipeline.yml

```diff
@@ -18,19 +18,18 @@ steps:
         required: false
         options:
           - label: "Continue without warning"
-            value: "skip"
+            value: "skiplinkcheck"
```

Comment:
Do we need to make corresponding changes to the `build.sh` script, which is looking for the shorter vales?

Reply:
The final looking change to those values will come from the `buildkite-webhook-rebuild-136` branch, and it's using the longer values. The values are just set automatically by the buildkite PR bot, not entered by folks in their PR.